### PR TITLE
add Io.GitHub.nokse22.inspector

### PR DESCRIPTION
**Pull request by Nokse22**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

I am using `--talk-name=org.freedesktop.Flatpak` because I need to run commands with `subprocess.Popen`

### CODE REVIEW

**Comment by Nokse22** [comment]
bot, build io.github.nokse22.inspector

File: io.github.nokse22.inspector.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "inspector",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=session-bus"
```
Comment (hfiguiere) [inline io.github.nokse22.inspector.json:12]: this is not allowed.

Comment (Nokse22) [reply io.github.nokse22.inspector.json:12]: I found out after adding it, i am trying to find out what permission i need be able to run commands with subprocess.Popen the app runs commands like `lsusb` and displays the results neatly.

`talk-name=org.freedesktop.Flatpak` worked in the app, but gave an error when built here.
   ```
 "errors": [
        "finish-args-flatpak-spawn-access"
```

Comment (JakobDev) [reply io.github.nokse22.inspector.json:12]: You need to [add a exception here](https://github.com/flathub/flatpak-builder-lint/blob/master/flatpak_builder_lint/staticfiles/exceptions.json).

btw:
We should block `--socket=session-bus` with flatpak-builder-lint by default.

Comment (Nokse22) [reply io.github.nokse22.inspector.json]: Thank you very much!

Comment (hfiguiere) [reply io.github.nokse22.inspector.json]: it is already blocked.

**Comment by Nokse22** [comment]
bot, build io.github.nokse22.inspector

**Comment by Nokse22** [comment]
Works as intended. Thank you very much, everybody!

**Comment by xduugu** [comment]
@barthalion @Nokse22 I noticed that this PR was based on the `master` branch and not `new-pr`. The result is that the app's repository contains several unrelated files and history. Maybe it would make sense to rebase the `master` branch and force-push it.

**Pull request closed without merging**